quil-rs/src/expression/mod.rs

```diff
@@ -1016,16 +1016,44 @@ mod tests {
             prop_assert_eq!(x, expected);
         }
 
+        // Redundant clone: clippy does not correctly introspect the prop_assert_eq! macro
```

Comment:
Nice find! I ended up splitting out a bunch of the proptests because of `clippy` arguing with me about clones.